spaceapi/src/lib.rs

```diff
@@ -82,6 +82,24 @@ impl<T> Optional<T> {
             Optional::Absent => Optional::Absent
         }
     }
+
+}
+
+/// An enum of all possible sensor templates.
+#[derive(Debug, Clone)]
+pub enum SensorTemplate {
+    PeopleNowPresentSensorTemplate {
+        location: Optional<String>,
+        name: Optional<String>,
+        names: Optional<Vec<String>>,
+        description: Optional<String>,
+    },
+    TemperatureSensorTemplate {
+        unit: String,
+        location: String,
+        name: Optional<String>,
+        description: Optional<String>,
+    },
```

Comment:
These are the partial sensor structs, so that we don't need to change anything about the `spaceapi` module, as @rnestler suggested. The data will be copied into the actual sensor struct.